plugins/ern_v0.28.0+/lottie-react-native_v2.5.11+/config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "android": {
+    "root": "src",
+    "moduleName": "android",
+    "dependencies": ["com.airbnb.android:lottie:2.6.0"]
```

Comment:
I see that they released 2.7.0 recently. Any reason to stay with 2.6.0?

Reply:
So this is depending on `lottie-react-native`, though it might be worth checking if they updated :) 
Will do so thanks!

Reply:
Yup the latest `lottie-react-native` is still `2.5.11`, and actually I put already a wrong version and it would be safer to go with the version here: https://github.com/react-native-community/lottie-react-native/blob/abf35622ebed2df274933c35be696229dfb36ba2/src/android/build.gradle#L30
Will change that.